tests/go_plus_sample.scm

```diff
@@ -0,0 +1,100 @@
+(EvaluationLink 
+	 (PredicateNode "has_name")
```

Comment:
Go names are not being found because the existing GO dataset predicate is "GO_name", I will fix the script, so that you will update the sample accordingly

Reply:
I have updated the dataset, use the latest